### PR TITLE
Conversion: Fixes #4141 dyne to newton conversion

### LINKED ISSUE #4141: Conversions: wrong results on dyne prefixes and dyne <-> newton conversion.
Following queries display wrong result :

"1 dyne in newton",
DuckDuckGo answered: 100,000 newtons. It should be 1 * 10^(-5) newtons.

"1 newton in dyne",
DuckDuckGo answered: 1 * 10^(-5) dynes. It should be 100,000 dynes.

"1 dyne in kilodyne",
DuckDuckGo answered: 1,000 kildynes. It should be 1 * 10^(-3) kilodynes.

"1 kilodyne in dyne",
DuckDuckGo answered: 0.00100 dynes. It should be 1000 dynes.

"1 dyne in megadyne" also gives wrong answer.

@adityatandon007 

------
IA Page:  http://duck.co/ia/view/conversions 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mintsoft

### PR DESCRIPTION
**Pull request by adityatandon007**
**Conversions IA**: Fixes dyne to newton and vice-versa conversion

## Description of new Instant Answer, or changes
This Pr fixes dyne newton conversion issue

## Related Issues and Discussions
Fixes #4141 

## People to notify
@pjhampton @mintsoft 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@adityatandon007 Can you add a test for this case?

File: share/goodie/conversions/ratios.yml [hunk 1]
```diff
@@ -1286,19 +1286,19 @@ symbols: [GN]
 plural: giganewtons
 ---
 aliases: []
-factor: 1e-05
+factor: 1e+05
```
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Please add a test for this

**Comment by moollaza** [comment]
Deploying to Beta 👍

**Comment by moollaza** [comment]
@adityatandon007 thanks for this! @pjhampton is fixing this in his Interactive Conversions PR now so I'll close this in favour of that PR which we plan to merge shortly 👍

**Pull request closed without merging**